src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -1089,6 +1089,68 @@ internal void SelectAllCategories(object parameter)
             }
         }
 
+        /// <summary>
```

Comment:
Hi, @ramramps , I can't say, that LINQ is much faster:
![image](https://cloud.githubusercontent.com/assets/8158404/12232000/4484a5ba-b864-11e5-9921-dbccf548d2f8.png)
![image](https://cloud.githubusercontent.com/assets/8158404/12232009/50ed410e-b864-11e5-8e43-434e98949818.png)
![image](https://cloud.githubusercontent.com/assets/8158404/12232013/5b305a20-b864-11e5-82ed-2fe48880bb52.png)
![image](https://cloud.githubusercontent.com/assets/8158404/12232019/654a4408-b864-11e5-944e-328c8a3a8a9e.png)
![image](https://cloud.githubusercontent.com/assets/8158404/12232026/7395082c-b864-11e5-9395-2819daacf4af.png)

But it definitely looks better :) So, let's leave LINQ here.